pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/player/SimplePlayerView.kt

```diff
@@ -83,22 +98,71 @@ fun SimplePlayerView(
     } else {
         modifier
     }
-    LocalView.current.keepScreenOn = player.isPlayingAsState()
-    DemoPlayerSurface(
-        modifier = surfaceModifier,
+    val isPlaying = player.isPlayingAsState()
+    LocalView.current.keepScreenOn = isPlaying
+    val interactionSource = remember {
+        MutableInteractionSource()
+    }
+    val isDragged = interactionSource.collectIsDraggedAsState().value
```

Comment:
⚠️ **[detekt]** <detekt.UnusedPrivateMember> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Private property `isDragged` is unused.